frontend/website-redesign/src/pages/Genesis.re

```diff
@@ -1,17 +1,263 @@
+            gridTemplateColumns([`rem(12.93), `rem(47.)]),
+            gridColumnGap(`rem(2.56)),
+            gridTemplateRows([`rem(4.), `rem(1.)]),
+            padding2(~v=`rem(2.5), ~h=`rem(2.5)),
+            height(`rem(11.)),
+          ],
+        ),
+      ]);
+    let culture =
+      style([
+        Theme.Typeface.monumentGrotesk,
+        color(white),
+        fontSize(`rem(2.)),
+      ]);
+    let body = merge([Theme.Type.paragraphSmall, style([color(white)])]);
+    let link =
+      style([
+        media(
+          Theme.MediaQuery.tablet,
+          [unsafe("grid-area", "2 / 2 / 3 /3 "), height(`rem(1.))],
+        ),
+        marginTop(`zero),
+      ]);
+  };
+  [@react.component]
+  let make = () => {
+    <Wrapped>
+      <div className=Styles.grid>
+        <h2 className=Styles.culture> {React.string("Our Culture")} </h2>
+        <p className=Styles.body>
+          {React.string(
+             "It's hard to quantify, but it's not hard to see: in any community, culture is everything.  Ours is rooted in respect and openness, curiosity and excellence.",
+           )}
+        </p>
+        <div className=Styles.link>
+          <Link
+            text="Read our Code of Conduct"
+            href="https://github.com/CodaProtocol/coda/blob/develop/CODE_OF_CONDUCT.md"
+          />
+        </div>
+      </div>
+      <Spacer height=1. />
+    </Wrapped>;
+  };
+};
+
 [@react.component]
 let make = () => {
   <Page title="Genesis Page">
-    <Wrapped>
-      <Hero
-        title="Community"
-        header="Genesis Program"
-        copy="We're looking for community members to join the Genesis Token Grant Program and form the backbone of Mina's robust decentralized network.">
-        <Spacer height=2. />
-        <Button bgColor=Theme.Colors.black>
-          {React.string("Apply Now")}
-          <Icon kind=Icon.ArrowRightMedium />
-        </Button>
-      </Hero>
-    </Wrapped>
+    <Hero
+      title="Community"
+      header="Genesis Program"
+      copy="We're looking for community members to join the Genesis Token Grant Program and form the backbone of Mina's robust decentralized network."
+      backgroundSmall="/static/img/GenesisSmall.jpg"
+      backgroundMedium="/static/img/GenesisMedium.jpg"
+      backgroundLarge="/static/img/GenesisLarge.jpg">
+      <Spacer height=2. />
+      <Button bgColor=Theme.Colors.black>
+        {React.string("Apply Now")}
+        <Icon kind=Icon.ArrowRightMedium />
+      </Button>
+    </Hero>
+    <FeaturedSingleRow
+      row={
+        FeaturedSingleRow.Row.rowType: ImageLeftCopyRight,
+        title: "Become a Genesis Member",
+        description: "Up to 1,000 community participants will be selected to help us harden Mina’s protocol, strengthen the network and receive a distribution of 66,000 tokens.",
+        textColor: Theme.Colors.white,
+        image: "/static/img/GenesisCopy.jpg",
+        background: Image("/static/img/BecomeAGenesisMemberBackground.png"),
```

Comment:
Since this is the same layout but the Spacing and background are different, maybe we can just remove the background image here and wrap the `FeaturedSingleRow` in an outer `div` that has the correct background and additional padding on the top that way the image doesn't overlap into the Hero.